lib/colorado_lottery.rb

```diff
@@ -0,0 +1,40 @@
+class ColoradoLottery
+
+  attr_reader :registered_contestants,
+              :winners,
+              :current_contestants,
+              :name
+
+  def initialize
+    @registered_contestants = {}
+    @winners = []
+    @current_contestants = {}
+  end
+
+  def interested_and_18?(contestant, game_name)
+    if contestant.game_interests != game_name.name
+      return false
+    elsif contestant.game_interests == game_name.name && contestant.age >= 18
+      true
+    end
+  end
+
+  def can_register?(contestant, game_name)
+    if contestant.age >= 18 || contestant.game_interests == game_name.name
+      return true
```

Comment:
this method had a bit more logic to it than you've included here-- a contestant had to be interested and 18, as well as an in-state resident or the lottery itself had to be national.   for the logic that you do have here, you could have reused your `interested_and_18` method. 

```
  def can_register?(contestant, game)
    state_resident = !contestant.out_of_state?
    national_game = game.national_drawing? == true

    interested_and_18?(contestant, game) && (state_resident || national_game)
  end
```